tests/helpers.js

```diff
@@ -104,6 +104,29 @@ export const endpointCheck = ({endpoint, vcVersion, keyType}) => {
   return true;
 };
 
+/**
+ * Filters verifiers using supports.vc and tags explicitly.
+ * Requires that function is bound to vcDefault, vcVersion, and tags.
+ *
+ * @param {object} options - Options to use.
+ * @param {object} options.implementation - An implementation.
+ *
+ * @returns {Array<object>} An array of filtered endpoints.
+ */
+export function filterVerifiers({implementation}) {
+  const endpoints = implementation.verifiers;
+  // the filter function expects an array to be returned
+  return endpoints.filter(e => {
+    // we want only endpoints that match every tag
+    if(this.tags.every(tag => e.tags.has(tag))) {
```

Comment:
Since `this` is handled via bind later, you should document that, so this function doesn't get called directly--which would have unexpected results to be sure.

Reply:
it's actually already in the jsdoc, but I guess I can clarify it further maybe.

Reply:
expanded the docs on `filterVerifiers.bind` here: https://github.com/w3c/vc-di-ecdsa-test-suite/pull/84/commits/bbf3a5f62f8c36501591162e9483a23748638e3e